src/Bindings/PluginManager.cpp

```diff
@@ -209,33 +202,45 @@ void cPluginManager::Tick(float a_Dt)
 		ReloadPluginsNow();
 	}
 
-	FIND_HOOK(HOOK_TICK);
-	if (Plugins != m_Hooks.end())
+	auto Plugins = m_Hooks.find(HOOK_TICK);
+	if (Plugins == m_Hooks.end())
 	{
-		for (PluginList::iterator itr = Plugins->second.begin(); itr != Plugins->second.end(); ++itr)
-		{
-			(*itr)->Tick(a_Dt);
-		}
+		return;
+	}
+
+	for (auto * Plugin : Plugins->second)
+	{
+		Plugin->Tick(a_Dt);
```

Comment:
Why doesn't this use the `GenericCallHook`?

Reply:
Because it can't. Tick doesn't return a bool like the other hooks.